### PR TITLE
Added npm package

### LINKED ISSUE #33: add package.json
would you please consider adding package.json so that we can add your lib on https://cdnjs.com.
thank you very much!
https://github.com/cdnjs/cdnjs/issues/5603

### PR DESCRIPTION
**Pull request by amoludage**
Fixed  #33 #32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ This plugin only provides client-side validation, and should be combined with so
 
 For more information, a demo, documentation, and the motivation behind Complexify, [visit the website](http://danpalmer.me/jquery-complexify).
 
+### Installation
+
+You can install this by using npm or bower
+```ruby
```
Comment (danpalmer) [inline README.md]: This should either be blank, or "shell"

File: package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "jquery.complexify",
+  "version": "0.5.1",
+  "main": "jquery.complexify.js",
+  "description": "Complexify helps you to accurately gauge the quality of a user's password to give them visual feedback, and to enforce a minimum level of security.",
+  "homepage": "https://github.com/amoludage/jquery.complexify.js",
```
Comment (danpalmer) [inline package.json]: This should link to the source repository.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  "keywords":[
+    "jquery",
+    "password",
+    "complexify"
+  ]
+
```
Comment (danpalmer) [inline package.json]: Unnecessary whitespace.

File: README.md (hunk 1 above)
Comment (amoludage) [inline README.md]: sorry I did not get this, what do you want?

File: package.json (hunk 2 above)
Comment (amoludage) [inline package.json]: ohh sorry, I will update this.

File: package.json (hunk 3 above)
Comment (amoludage) [inline package.json]: I will remove this spacing

**Comment by amoludage** [comment]
@danpalmer Updated PR :+1:  please review & let me know if you found any issue. Thanks

**Comment by danpalmer** [comment]
Great, thanks for doing this, and sorry it took me so long to get around to reviewing/merging. Much appreciated!

**Comment by danpalmer** [comment]
@amoludage could you please transfer ownership of the NPM package to me?

**Comment by amoludage** [comment]
@danpalmer I have added you to the contributor of  [jquery.complexify.js](https://www.npmjs.com/package/jquery.complexify).

Sorry I don't know how to transfer ownership of packages :relaxed: 

If you know about how to transfer the ownership of `npm`  packages, then please share transferring steps with me so that I can transfer it.

Thanks.

**Comment by jsumners** [comment]
https://docs.npmjs.com/cli/owner

```
$ npm owner add new@owner
$ npm owner rm old@owner
```
